src/utils/sharedFunctions.ts

```diff
@@ -2,16 +2,15 @@
 // The total can be passed directly in as a number or a string containing a number, or can
 // be derived by adding up 1) the values in an array of numbers, 2) the values in an array of strings containing numbers,
 // or 3) the lengths of the elements in an array of arrays.
-export function getPercentage(x: number, outOf: number | string | Array<string | number | Array<any>>, asString: boolean = false) {
+export function getPercentage(x: number, outOf: number | string | Array<string | number>, asString: boolean = false) {
   let total;
   if (Array.isArray(outOf)) {
-    total = outOf.map((el) => {
-      return Array.isArray(el) ? el.length : parseInt(el as string);
-    }).reduce((accum, next) => accum + next);
+    total = outOf.map((el) => { return parseInt(el as string); }).reduce((accum, next) => accum + next);
```

Comment:
Nothing serious but you can do `map((el) => parseInt(el as string))` to be consistent with the reduce callback function.